frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/create-obc.tsx

```diff
@@ -21,6 +21,9 @@ import { ActionGroup, Button } from '@patternfly/react-core';
 import { getName } from '@console/shared';
 import { commonReducer, defaultState } from '../object-bucket-page/state';
 
+// Accept only noobaa provisioned
+const filterStorageClasses = (sc) => sc.provisioner === 'openshift-storage.noobaa.io/obc';
```

Comment:
create a const for `noobaa.io/obc`
In generic utils of noobaa right the fuction to filter and don't consider `openshift-storage` prefix like we did for ceph.
```suggestion
const filterStorageClasses = (sc) => sc.provisioner === 'openshift-storage.noobaa.io/obc';
```

Reply:
okay

Reply:
I moved it to utils. But does it make sense to make a const while it's only being used in one place?